dom/src/mockDOMSource.ts

```diff
@@ -1,81 +1,81 @@
   [name: string]: GenericStream | MockConfig;
   elements?: GenericStream;
 }
 
-const SCOPE_PREFIX = '___';
+const CYCLE_SOURCE = `MockedDOM`;
+const CSS_NOTATION = `.`;
+const SCOPE_PREFIX = `___`;
 
-export class MockedDOMSource implements DOMSource {
+export class MockedDOMSource implements DOMSourceable {
   private _elements: any;
 
   constructor(private _streamAdapter: StreamAdapter,
               private _mockConfig: MockConfig) {
-    if (_mockConfig.elements) {
-      this._elements = _mockConfig.elements;
-    } else {
-      this._elements = _streamAdapter.adapt(xs.empty(), xsSA.streamSubscribe);
-    }
+    this._elements = _mockConfig.elements ||
+                     _streamAdapter.adapt(xs.empty(), xsAdapter.streamSubscribe);
   }
 
   public elements(): any {
     const out: DevToolEnabledSource = this._elements;
-    out._isCycleSource = 'MockedDOM';
+    out._isCycleSource = CYCLE_SOURCE;
+
     return out;
   }
 
   public events(eventType: string, options: EventsFnOptions): any {
     const mockConfig = this._mockConfig;
     const keys = Object.keys(mockConfig);
-    const keysLen = keys.length;
-    for (let i = 0; i < keysLen; i++) {
-      const key = keys[i];
+    const keysCount = keys.length;
+    for (let idx = 0; idx < keysCount; idx++) {
+      const key = keys[idx];
       if (key === eventType) {
         const out: DevToolEnabledSource = mockConfig[key];
-        out._isCycleSource = 'MockedDOM';
+        out._isCycleSource = CYCLE_SOURCE;
+
         return out;
       }
     }
     const out: DevToolEnabledSource = this._streamAdapter.adapt(
       xs.empty(),
-      xsSA.streamSubscribe
+      xsAdapter.streamSubscribe
     );
-    out._isCycleSource = 'MockedDOM';
+    out._isCycleSource = CYCLE_SOURCE;
+
     return out;
   }
 
-  public select(selector: string): DOMSource {
+  public select(selector: string): DOMSourceable {
     const mockConfig = this._mockConfig;
     const keys = Object.keys(mockConfig);
-    const keysLen = keys.length;
-    for (let i = 0; i < keysLen; i++) {
-      const key = keys[i];
+    const keysCount = keys.length;
+    for (let idx = 0; idx < keysCount; idx++) {
+      const key = keys[idx];
       if (key === selector) {
         return new MockedDOMSource(this._streamAdapter, mockConfig[key]);
       }
     }
+
     return new MockedDOMSource(this._streamAdapter, {});
   }
 
-  public isolateSource(source: MockedDOMSource, scope: string): DOMSource {
-    return source.select('.' + SCOPE_PREFIX + scope);
+  public isolateSource(source: MockedDOMSource, scope: string): DOMSourceable {
+    return source.select(CSS_NOTATION + SCOPE_PREFIX + scope);
   }
 
   public isolateSink(sink: any, scope: string): any {
     return sink.map((vnode: VNode) => {
       if (vnode.sel.indexOf(SCOPE_PREFIX + scope) !== -1) {
         return vnode;
       } else {
-        vnode.sel += `.${SCOPE_PREFIX}${scope}`;
+        vnode.sel += `${CSS_NOTATION}${SCOPE_PREFIX}${scope}`;
```

Comment:
This CSS_NOTATION stuff is not helping. If I want to understand how this line works, I have to look up what on earth is CSS_NOTATION. `.${SCOPE_PREFIX}${scope}` was more readable without looking up anything somewhere. As frontend developers, by now we should be entirely familiar with `.foo` as a classname selector.